lib/benefits/compute-javascript.ts

```diff
@@ -282,6 +282,33 @@ export const CONDITION_STRATEGY: Conditions = {
       return OPERATOR[condition.operator](taux_incapacite, condition.value)
     },
   },
+  enfants_eligibles: {
+    test: (condition, { situation }: { situation: Situation }) => {
+      if (!situation.enfants || situation.enfants.length === 0) {
+        return false
+      }
+
+      let enfantsEligibles = situation.enfants
+      if (condition.age_min !== undefined) {
```

Comment:
Il faudrait plutôt faire : 
```suggestion
      if (condition.age_min !== undefined || condition.age_max !== undefined) {
```